main_dialog.py

```diff
@@ -0,0 +1,308 @@
+            # same extensions.\nEnable checkBox")   # for debugging
+
+            self.check_box_1.setEnabled(True)
+        self.check_box_1.setEnabled(True)
+        return True
+
+    def items_changed(self):
+        if(self.selectedRecipe is None and defaultRecipe is None or
+           self.list_widget_1.count() == 0):
+
+            self.push_button_3.setEnabled(False)
+            self.check_extensions()
+        else:
+            self.push_button_3.setEnabled(True)
+            self.check_extensions()
+
+
+# clear all files on the list
+    def clear_all_items(self):
+        self.list_widget_1.clear()
+        self.items_changed()
+
+    # select all files on the list
+    def select_items(self):
+        self.list_widget_1.selectAll()
+        # print("\n" + str(listPaths)) # chcek list values: listPaths
+        for x in range(self.list_widget_1.count()):
+            print(self.list_widget_1.item(x).showPath())
+
+    def print_extensions(self):
+        for x in range(self.list_widget_1.count()):
+            print(self.list_widget_1.item(x).showExtension())
+
+    def dragEnterEvent(self, e):
+        e.accept()
+
+    def dropEvent(self, e):
+        self.items_changed()
+
+# << ADD paths on list_widget_1 from listPaths used pop()  >> #
+    def add_to_list_widget(self, listPaths):
+        for path in listPaths:
+            fileName = str(path).split("/")
+            item = MyListWidgetItem(path)
+            item.setText(fileName[-1])
+            item.setPath(str(path))
+            self.list_widget_1.addItem(item)
+        # for debugging
+        self.print_extensions()
+        self.check_extensions()
+
+# << Load files on >> #
+
+    def load_files(self):
+        listPaths = []
+        files, _ = QFileDialog.getOpenFileNames(
+                self, "Wybierz pliki", '',
+                "wszystkie (*);; " + "commonmark (*.commonmark);;" +
+                "docbook (*.docbook);;" + "docx (*.docx);;" +
+                "epub (*.epub);;" + "haddock (*.haddock);;" +
+                "html (*.html);;" + "json (*.json);;" + "latex (*.latex);;" +
+                "markdown (*.markdown *.md);;" +
+                "markdown_github (*.markdown_github);;" +
+                "markdown_mmd(*.markdown_mmd);;" +
+                "markdown_phpextra (*.markdown_phpextra);;" +
+                "markdown_strict (*.markdown_strict);;" +
+                "mediawiki (*.mediawiki);;" + "native (*.native);;" +
+                "odt (*.odt);;" + "opml (*.opml);;" + "org (*.org);;" +
+                "rst (*.rst);;" + "t2t (*.t2t);;" + "textile (*.textile);;" +
+                "twiki (*.twiki)")
+
+        for file in files:
+            listPaths.append(file)
+        print(listPaths)  # for debugging
+        self.add_to_list_widget(listPaths)
+        self.items_changed()
+# << END of: Load files on >> #
+
+    def return_boxIsChecked(self):
+        self.isChecked = self.check_box_1.checkState()
+        return self.isChecked
```

Comment:
Po pierwsze czasem jest camelCase a czasem podkreślenia - brak spójności. Po drugie, można nazwać po prostu `isBoxChecked`, raczej return się nie stosuje w nazwie. Zauważ również, że to jest taki setterogetter bo funkcja nie tylko zwraca wartość, ale też uaktualnia polę klasy. Wydzieliłbym z funkcji drugą `updateBoxChecked`  i zmienił nazwę pola z `isChecked` na `boxChecked`, bo nie jest ona do końca szczęśliwa.